CoffeeMachineSimulator/CoffeeMachineSimulator.Tests/Services/CoffeeServiceTests.cs

```diff
@@ -36,8 +36,50 @@ public void AddNullCoffee_Throws_NewException()
             
             Assert.AreEqual(ex.Message, "You should not add null entries!");
         }
-
+        
+        [Test]
         //TODO: Add Unit test for Remaining add cases
+        public void AddCoffeWithoutPrice_DoesNotAddCoffee()
+        {
+            var LengthList = coffeeService.GetCoffees().Count;
```

Comment:
please use camelCase when naming variables and be consistent with them. Also, the naming of the LengthList variable is a bit ambigous, please rename it